docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md

```diff
@@ -64,18 +64,18 @@ Examples of how a text range is adjusted for Move() and ExpandToEnclosingUnit()
 ### Hyperlink  
  **Example 1 - A text range that contains an embedded text hyperlink**  
   
- {The URL [http://www.microsoft.com](https://www.microsoft.com) is embedded in text}.  
+ {The URL <https://www.microsoft.com> is embedded in text}.  
```

Comment:
Wondering if we should put this entire sentence in backticks and remove the URL formatting.

Reply:
why there more than at line 42?

Reply:
My suggestion is because this is supposed to be plain text here

Reply:
So, I remove the enclosing <> of the urls and put the whole lines in backticks?